include/laser_assembler/base_assembler.h

```diff
@@ -194,27 +197,27 @@ void BaseAssembler<T>::start(const std::string& in_topic_name)
 {
   ROS_DEBUG("Called start(string). Starting to listen on message_filter::Subscriber the input stream");
   if (tf_filter_)
-    ROS_ERROR("assembler::start() was called twice!. This is bad, and could leak memory") ;
+    ROS_WARN("assembler::start() was called twice!. This is bad, and could leak memory if not done right") ;
```

Comment:
The way I called these functions and given their implementation, I think it's fine and there is no memory leak. Did not measure to verify